e2e/mock-api/src/app/routes.resource.js

```diff
@@ -115,7 +120,13 @@ export default function (app) {
         baz.canWithdraw = false;
         res.json({ message: 'Successfully retrieved resource!' });
       } else {
-        res.redirect(307, createStepUpUrl(req.headers.referer));
+        if (req.headers['x-forgerock-sdk']) {
+          res.setHeader('WWW-Authenticate', createStepUpHeader(req.headers.referer));
+          res.setHeader('Access-Control-Expose-Headers', 'www-authenticate');
```

Comment:
I had to add this in order for the test app to be able to view the WWW-Authenticate header on 401 response - otherwise I seemed to have no access to headers, which was weird. IG provides this header by the way, so I thought it was fine to include here.

Reply:
This should likely go into the CORS configuration here: https://github.com/ForgeRock/forgerock-javascript-sdk/blob/develop/e2e/mock-api/src/index.js#L31. Anything related to `Access-Control` is a CORS concern.

Reply:
Docs for the CORS middleware can be found here: https://expressjs.com/en/resources/middleware/cors.html

Reply:
That makes a lot more sense. Thanks!